en/releases/1.12.md

```diff
@@ -5,26 +5,65 @@
 
 ## Changes
 
+* [Common](#common)
+* [MAVLink](#mavlink)
+* [Commander](#commander)
 * [Multicoper](#multicopter)
+* [VTOL](#vtol)
+* [Control & Navigation](#control)
+
+### Common
+
+* **RTL Trigger based on remaining flight range ([PR#16399](https://github.com/PX4/PX4-Autopilot/pull/16399))**
+  * Calculates time to home, on RTL, taking into account vehicle speed, wind speed, and destination distance/direction
+* **Pre-emptive geofence breach ([PR#16400](https://github.com/PX4/PX4-Autopilot/pull/16400))**
+  * Triggers a breach if the _predicted_ current trajectory will result in a breach, allowing the vehicle to be re-routed to a safe hold position.
+
+### MAVLink
+
+* **MAVLink Ethernet configuration ([PR#14460](https://github.com/PX4/PX4-Autopilot/pull/14460))**
+  * MAVLink Ethernet channel settings such as UDP port, remote port and broadcast mode now can be changed dynamically via parameters.
+* **Support for querying `COMPONENT_INFORMATION` ([PR#16039](https://github.com/PX4/PX4-Autopilot/pull/16039))**
+  * This new message allows a GCS to request rich hierarchical information from an autopilot; i.e. in order to understand which commands are supported in missions, or to get parameter metadata (RFC: [mavlink#1339](https://github.com/mavlink/mavlink/issues/1339))
+
+### Commander
+
+* **Commander: use control mode flags and cleanup arm/disarm ([PR#16266](https://github.com/PX4/PX4-Autopilot/pull/16266))**
+  * Consolidates arming logic, and does a better job at keeping the current state
+* **Commander: Separate out manual control setpoint processing ([PR#16878](https://github.com/PX4/PX4-Autopilot/pull/16878))**
+  * Adds a new class `ManualControl` for handling `manual_control_setpoint` and handles RC loss, RC override, and RC arming/disarming
```

Comment:
Same question, is there any impact on end uses, any docs impact? (this is OK, just a query)

Reply:
Yeah there are consequences here, and should be documented on the guides.

Reply:
https://github.com/PX4/PX4-user_guide/issues/1137

Reply:
@mrpollo FYI following a bit of research and discussion with Mattias there isn't any docs consequence that was not already in PX4 v1.11. I did some tidy on that. Closing.